github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -789,7 +789,7 @@ presubmits:
           path: /dev/vfio/
           type: Directory
         name: vfio
-  - always_run: true
+  - always_run: false
```

Comment:
Why do you keep the old one? Seems to me like it can be removed, no?

Reply:
Yes it can, but I'm not sure about the lifecycle of the lanes (if they must exists for a certain period or something like that). If you say that I can delete it I can do it. :slightly_smiling_face:

Reply:
IMO it can be removed, usually release branches use different jobs.

@dhiller , WDYT?

Reply:
In general the lifecycle of a lane seen for itself is:

- introduce lane with always_run: false, optional:true, skip_report: true
- collect some data
- update lane to always_run: true, optional:true, skip_report: true
- collect some data
- update lane to final status if everything is ok, this would be removing optional and skip_report 

Not sure whether we have enough data to decide whether the lane is stable enough and to delete the old lane. WDYT?

https://prow.ci.kubevirt.io/?repo=kubevirt%2Fkubevirt&job=*-kind-1.23-vgpu

Reply:
Thank you @dhiller! I will change the PR to `always_run: true, optional:true, skip_report: true` in order to collect more data.

Reply:
@fossedihelm sounds good to me! Thanks!